components/shared/searchFilters/helper.ts

```diff
@@ -0,0 +1,49 @@
+import { FacetField, SearchFilters } from "@/lib/graphql/generated/fbi/graphql"
+import { AppRouterInstance } from "next/dist/shared/lib/app-router-context.shared-runtime"
+
+export const facetDefinitions = [
+  "materialTypesGeneral",
+  "mainLanguages",
+  "age",
+  "lix",
+  "subjects",
+] as FacetField[]
+
+export const mapFacetsToFilters = {
+  materialTypesGeneral: "materialTypesGeneral",
+  mainLanguages: "mainLanguages",
+  age: "age",
+  lix: "lixRange",
+  subjects: "subjects",
+} as Record<FacetField, keyof SearchFilters>
+
+export const mapFilterNameToTranslation: Partial<Record<keyof typeof mapFacetsToFilters, string>> =
+  {
+    materialTypesGeneral: "Type",
+    mainLanguages: "Sprog",
+    age: "Alder",
+    lix: "Lix",
+    subjects: "Emne",
+  }
```

Comment:
We should handle these facets/filters in a cetralized configuration manner. But let's talk about it and a `TODO`? would be in place...

Reply:
I'm not sure if these are a part of configuration. I don't think they ever should be configurable.